src/Configurator/DockerComposeConfigurator.php

```diff
@@ -125,6 +138,45 @@ private function normalizeConfig(array $config): array
         return $config;
     }
 
+    /**
+     * Finds the Docker Compose file according to these rules: https://docs.docker.com/compose/reference/envvars/#compose_file.
+     */
+    private function findDockerComposeFile(string $rootDir, string $file): ?string
+    {
+        if (isset($_SERVER['COMPOSE_FILE'])) {
+            $files = explode('\\' === \DIRECTORY_SEPARATOR ? ';' : ':', $_SERVER['COMPOSE_FILE']);
+            foreach ($files as $f) {
+                if ($file !== basename($f)) {
+                    continue;
+                }
+
+                if (!$this->filesystem->isAbsolutePath($f)) {
+                    $f = realpath(sprintf('%s/%s', $rootDir, $f));
+                }
+
+                return $this->filesystem->exists($f) ? $f : null;
```

Comment:
while returning null otherwise? shouldn't we fallback to other files defined in the `COMPOSE_FILE` variables?

Reply:
Indeed we can do that. I'll change this.